### PR TITLE
Small CLI Updates

### PR DESCRIPTION
**Pull request by cliveseldon**
- [x] Allow a `--truncate` option to not show data in `seldon inspect` as sometimes data can be very large and you want to just see outline of message
- [x] Allow `seldon pipeline unload` to take a `-f <filename>` like load and similar to k8s kubectl
- [x] Allow `seldon model unload` to take a `-f <filename>`
- [x] Allow `seldon experiment unload` to take a `-f <filename>`

### CODE REVIEW

**Review by sakoush — COMMENTED** [review]
Do we already allow  models / experiments to be unloaded similarly?

**Comment by cliveseldon** [comment]
> Do we already allow models / experiments to be unloaded similarly?

Updated to include models and experiments

File: operator/cmd/seldon/cli/model_unload.go [hunk 1]
```diff
@@ -49,14 +48,25 @@ func createModelUnload() *cobra.Command {
...
+				return err
+			}
+			var fileBytes []byte
+			if filename != "" {
+				fileBytes = loadFile(filename)
+			}
+			modelName := ""
+			if len(args) > 0 {
+				modelName = args[0]
+			}
```
Comment (sakoush) [inline operator/cmd/seldon/cli/model_unload.go:62]: these blocks of code look that they can be extracted as a helper for the 3 use cases (pipeline / model / experiment).

**Review by sakoush — APPROVED** [review]
LGTM. I left a refactor comment but logic is ok.

**Pull request merged**